core/src/main/java/hudson/tasks/_maven/MavenWarningNote.java

```diff
@@ -54,5 +55,6 @@ public String getDisplayName() {
         }
     }
 
+    @SuppressFBWarnings("MS_SHOULD_BE_FINAL")
     public static Pattern PATTERN = Pattern.compile("^\\[WARNING\\]");
```

Comment:
Taking the previous changes, should be final as well to keep it consistent

Reply:
You are right. added final and removed suppression